src/inc/msquic.h

```diff
@@ -473,6 +473,35 @@ typedef enum QUIC_PERFORMANCE_COUNTERS {
     QUIC_PERF_COUNTER_MAX,
 } QUIC_PERFORMANCE_COUNTERS;
 
+typedef struct QUIC_VERSION_SETTINGS {
+    union {
+        uint64_t IsSetFlags;
+        struct {
+            uint64_t DesiredVersionsList                    : 1;
+            uint64_t VersionNegotiationExtEnabled           : 1;
+            uint64_t RESERVED                               : 62;
+        } IsSet;
+    };
+
+    const uint32_t* DesiredVersionsList;
+    uint32_t DesiredVersionsListLength;
+    uint8_t VersionNegotiationExtEnabled    : 1;
+    uint8_t RESERVED                        : 7;
+} QUIC_VERSION_SETTINGS;
```

Comment:
I was thinking about that last night, and I'll just move VN Ext enabled back to the main settings struct.

Reply:
With the changes for the VN draft, we probably want to make version negotiation a get/set param instead of a settings thing. Pending further discussion.